### ISSUE TITLE
very low mAP on the training data of small dataset

### ISSUE THREAD

**Issue by Noebarrell** [jwyang/faster-rcnn.pytorch#580]
To verify that everything is functioning correctly, I am training on a very small dataset of approximately 50 images. _As a sanity check_, I want to see this model get a high mAP on the training data, however even after the loss converges to 0.0015, it is not able to achieve a mAP higher than 0.03 on the training set. The data I'm using is my own custom dataset. It seems like something else is wrong, because on such a small, consistently annotated, 2-class dataset it should easily be able to achieve a very high training precision. Does anyone have any ideas what could be going on? 

I simply followed the advice of #221 and simply placed the data in the pascal_voc format, and changed the class labels in the data loader files

**Comment by JigyasaK** [comment]
How did you solve this problem?
